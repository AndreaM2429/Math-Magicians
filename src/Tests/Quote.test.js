// import react-testing methods
import renderer from 'react-test-renderer';
// the component to test
import QuoteApi from '../Components/Quote_API';

test('testing layout', async () => {
  const tree = renderer.create(<QuoteApi />).toJSON();
  expect(tree).toMatchSnapshot();
});
