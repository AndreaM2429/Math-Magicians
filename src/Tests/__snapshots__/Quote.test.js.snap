// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing layout 1`] = `
<section
  className="quotes d-flex"
>
  <p
    title="text-quote"
  />
  <p
    className="author"
  />
</section>
`;
